coalaip_bigchaindb/plugin.py

```diff
@@ -109,14 +115,24 @@ def save(self, entity_data, *, user):
         """
 
         try:
-            tx_json = self.driver.transactions.create(
-                entity_data,
-                verifying_key=user['verifying_key'],
-                signing_key=user['signing_key'])
+            tx = self.driver.transactions.prepare(
+                operation='CREATE',
+                owners_before=user['verifying_key'],
+                # TODO: Where to put this "normalization"?
+                asset={'data': entity_data})
         except BigchaindbException as ex:
             raise EntityCreationError(error=ex)
+        try:
+            fulfilled_tx = self.driver.transactions.fulfill(
+                tx, private_keys=user['signing_key'])
+        except MissingSigningKeyError as ex:
```

Comment:
Cool 👍, just wasn't sure based on the name of the exception since we're theoretically *always* giving the signing key (it just might not be the correct one).